src/example-types/tagsQuery.js

```diff
@@ -0,0 +1,89 @@
+let _ = require('lodash/fp')
+let { parens } = require('futil-js')
+let Combinatorics = require('js-combinatorics')
+
+let wordPermutations = word =>
+  _.map(_.join(' '), Combinatorics.permutation(word.split(' ')).toArray())
+
+let limitResultsToCertainTags = tags => !!_.find('onlyShowTheseResults', tags)
+
+let wrapIf = _.curry(
+  (pre, post, text, shouldWrap) => (shouldWrap ? `${pre}${text}${post}` : text)
+)
+let quoteIf = wrapIf('"', '"')
```

Comment:
I think these wrapIf/quoteIf things probably should be removed - I think I was the original author of it and it's still kinda funky to me. F.when/F.unless can work if you don't want the tertiary stuff